pkg/cli/admin/createkubeconfig/create_kubeconfig.go

```diff
@@ -93,6 +93,8 @@ func NewCommandCreateKubeConfig(commandName string, fullName string, streams gen
 				kcmdutil.CheckErr(err)
 			}
 		},
+		Deprecated: "and will be removed in the future version",
+		Hidden:     true,
```

Comment:
Nit: this is not needed, each deprecated command is automatically hidden.

Reply:
oops and updated